.release-please-manifest.json

```diff
@@ -0,0 +1 @@
+{ "ConfidenceProvider": "0.1.0" }
```

Comment:
There is no folder called `ConfidenceProvider`, are you sure about this?

Reply:
it's in the [sources ](https://github.com/spotify/confidence-openfeature-provider-swift/tree/main/Sources/ConfidenceProvider). i'm not sure if it picks it up like that.